File: CampusServer/src/main/java/com/cai/campus/server/entity/UserAccount.java
package com.cai.campus.server.entity;

import java.io.Serializable;

/**
 * (UserAccount)实体类
 *
 * @author makejava
 * @since 2020-03-14 17:42:05
 */
public class UserAccount implements Serializable {
    private static final long serialVersionUID = 573386372958693835L;

    private Integer uid;

    private String phone;

    private String password;

    private String name;

    private Integer sex;

    private String logo;

    private Integer birthday;

    private String introduction;

    private String email;


    public Integer getUid() {
        return uid;
    }

    public void setUid(Integer uid) {
        this.uid = uid;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getSex() {
        return sex;
    }

    public void setSex(Integer sex) {
        this.sex = sex;
    }

    public String getLogo() {
        return logo;
    }

    public void setLogo(String logo) {
        this.logo = logo;
    }

    public Integer getBirthday() {
        return birthday;
    }

    public void setBirthday(Integer birthday) {
        this.birthday = birthday;
    }

    public String getIntroduction() {
        return introduction;
    }

    public void setIntroduction(String introduction) {
        this.introduction = introduction;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

}

File: CampusClient/app/src/main/java/com/cai/campus/common/network/model/UserAccount.kt
package com.cai.campus.common.network.model

import java.io.Serializable

class UserAccount : Serializable {
    constructor() {}
    constructor(phone: String?, password: String?) {
        this.phone = phone
        this.password = password
    }

    var uid: Int? = null
    var phone: String? = null
    var password: String? = null
    var name: String? = null
    var sex: Int? = null
    var logo: String? = null
    var birthday: Int? = null
    var introduction: String? = null
    var email: String? = null

    companion object {
        private const val serialVersionUID = 573386372958693835L
    }
}